pool/chainstate.go

```diff
@@ -119,7 +120,18 @@ func (cs *ChainState) pruneAcceptedWork(ctx context.Context, height uint32) erro
 		}
 		confs, err := cs.cfg.GetBlockConfirmations(ctx, hash)
 		if err != nil {
-			return err
+			var rErr *dcrjson.RPCError
+			switch errors.As(err, &rErr) {
+			case true:
+				// Do not error if the block being pruned cannot be found. It
+				// most likely got reorged off the chain.
+				if rErr.Code != dcrjson.ErrRPCBlockNotFound {
```

Comment:
Not working, I am still seeing the same error as before. I think this needs to be changed:

```suggestion
				if err.Code != dcrjson.ErrRPCBlockNotFound {
```